pacstall

```diff
@@ -179,140 +184,164 @@ while [[ "$1" =~ ^- && ! "$1" == "--" ]]; do
       -V --version, Prints pacstall version
       -L --list, Lists installed packages
       -Up --upgrade, Upgrades packages
-      -Qd --query-depends, Get dependencies of package
       -Qi --query-info, Get package info
        "
       exit 0
-      ;;
+    ;;
+
     -I|--install)
-      export LC_ALL=C
       function trap_ctrlc () {
-	    fancy_message warn "the installation of $2 was interrupted, removing files"
-	    rm -rf /tmp/pacstall/*
+	      fancy_message warn "the installation of $2 was interrupted, removing files"
+	      rm -rf "${SRCDIR:?}"/* # :? makes bash error out in case SRCDIR is empty, saving us from yonking /* directory by mistake
         exit 2
       }
       trap "trap_ctrlc" 2
+
       PACKAGE=$2
       if [[ -z "$PACKAGE" ]]; then
         fancy_message error "You failed to specify a package"
         exit 1
       fi
+
       # Make the directory if not exist
-      if [[ ! -e /usr/share/pacstall/repo/ ]]; then
-        sudo mkdir -p /usr/share/pacstall/repo
-        sudo touch /usr/share/pacstall/repo/pacstallrepo.txt
-	    sudo pacstall -C
+      if [[ ! -e "$STGDIR/repo/" ]]; then
+        sudo mkdir -p "$STGDIR/repo"
+        sudo touch "$STGDIR/repo/pacstallrepo.txt"
+	      sudo pacstall -C
```

Comment:
Shouldn't this be spaced equally with the rest of the if statement?

Reply:
Hmm.. It's fine in my editor?
![img](https://0x0.st/-91t.png)

Reply:
well, it isn't on github so...

Reply:
```suggestion
	  sudo pacstall -C
```
So this then?

Reply:
I think so. I can't see the context of the change compared to the rest of the code around it